region/norfair/west/Crocomire Speedway.json

```diff
@@ -592,6 +886,47 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 5],
+      "name": "Come In Shinecharging, Leave With Spark",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 1,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canShinechargeMovementTricky",
+        "canInsaneJump",
+        "canBePatient",
+        {"enemyDamage": {
```

Comment:
Its a bit easier if you kill the cac - I was using plasma.

Reply:
yes, it only felt like it saved 2 or 3 frames.  There are still 2 other very precise jumps.